website/docs/internals/json-format.html.md

```diff
@@ -0,0 +1,481 @@
+---
+layout: "docs"
+page_title: "Internals: JSON Output Format"
+sidebar_current: "docs-internals-json"
+description: |-
+  Terraform provides a machine-readable JSON representation of state, configuration and plan.
+---
+
+# JSON Output Format
+
+-> **Note:** This format is available in Terraform 0.12 and later.
+
+When Terraform plans changes, it prints a human-readable summary to the terminal. It can also, when run with `-out=<PATH>`, write a much more detailed binary plan file, which can later be used to apply those changes.
```

Comment:
```suggestion
When Terraform plans to make changes, it prints a human-readable summary to the terminal. It can also, when run with `-out=<PATH>`, write a much more detailed binary plan file, which can later be used to apply those changes.
```

...reading this again in the morning, my brain glitched out a bit bc it was expecting "when plans change;" this should make it flow a little better.